docs/dev-tools/console/console.asciidoc

```diff
@@ -65,9 +63,68 @@ These suggestions show you the parameters for each API and speed up your typing.
 To configure your preferences for autocomplete, go to
 <<configuring-console, Settings>>.
 
+[float]
+[[console-comments]]
+==== Comments
+
+You can write notes or temporarily disable parts of a request by using double forward slashes
```

Comment:
```suggestion
You can write comments or temporarily disable parts of a request by using double forward slashes
```

Reply:
In code we refer more to "comments" than "notes"?

Reply:
Oh I see. I was thinking that the feature is called "Comments", and that "notes" and "disabling parts of a request" are two use cases for this feature. Do you think this makes sense? Do you think there's a better way to communicate this idea?

Reply:
I would use "comments" to match the title of this section.  

Do you want to include what the symbols look like? (//) and (#)?

Reply:
Maybe also include an example with a pound sign?

Reply:
OK thanks both, I've updated this to "comments".

@gchaps Good suggestion. The first line of the code example has a pound sign:

```
# This request searches all of your indices.
GET /_search
{
  // The query parameter indicates query context.
  "query": {
    "match_all": {} // Matches all documents.
  }
}
```